mediachain/datastore/utils.py

```diff
@@ -0,0 +1,66 @@
+import base58
+import cbor
+from mediachain.proto import Types_pb2  # pylint: disable=no-name-in-module
+from mediachain.datastore.data_objects import MultihashReference
+
+
+def multihash_ref(ref):
+    if isinstance(ref, MultihashReference):
+        return ref
+
+    try:
```

Comment:
this style of flow control makes me super uncomfortable, but I guess it's pretty pythonic?